### ISSUE TITLE
remove h3 and replace with label 

### ISSUE THREAD

**Issue by CapSap** [CapSap/whose-drink#5]
app fails accessibility- I need to remove nesting of label and input, replace h3 with label. 
![image](https://user-images.githubusercontent.com/23469082/147468198-4598adfa-ea16-4718-a6a0-b9d958657412.png)
![image](https://user-images.githubusercontent.com/23469082/147468219-0d2babc3-1a8e-42b8-9037-1289f5fb2920.png)
